retriever/bin/retriever.js

```diff
@@ -16,17 +16,11 @@ export default {
    * @param {ExecutionContext} ctx
    * @param {object} options
    * @param {typeof defaultRetrieveFile} [options.retrieveFile]
-   * @param {AbortSignal} [options.signal]
```

Comment:
The abort signal had to be removed, the proper way is to listen to `request.signal`, see https://developers.cloudflare.com/changelog/2025-05-22-handle-request-cancellation/